include/glow/Support/Support.h

```diff
@@ -68,6 +68,19 @@ inline std::string tolower(const std::string &in) {
   return data;
 }
 
+/// \return the color based on \p index which is used in dot file.
+inline const char *getDotFileColor(size_t index) {
```

Comment:
And I'd suggest moving the implementation into the cpp file.

Reply:
Done.